sklearn/tests/test_grid_search.py

```diff
@@ -429,6 +429,7 @@ def predict(self, X):
         return np.zeros(X.shape[0])
 
 
+@ignore_warnings
```

Comment:
`BrokenClassifier` predicts only zeros, so the precision is not defined (zero division), returns zero and raises a warning. Once again, I think we can silence this warning.